src/components/BottomSheet/BottomSheetMenu.js

```diff
@@ -34,13 +34,13 @@ export default BottomSheetMenu;
 
 const styles = StyleSheet.create({
   rbsheetContainer: (height) => ({
-    backgroundColor: colors.white,
+    backgroundColor: COLORS.white,
     borderTopLeftRadius: dimen.normalizeDimen(12),
     borderTopRightRadius: dimen.normalizeDimen(12),
     height: dimen.normalizeDimen(height)
   }),
   rbsheetDraggableIcon: {
-    backgroundColor: colors.greySubtile1,
+    backgroundColor: COLORS.greySubtile1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `backgroundColor` property in the `rbsheetDraggableIcon` style has been updated to use the new `COLORS` object. This change is consistent with the PR's goal of standardizing color usage across the application.
```diff
-    backgroundColor: colors.greySubtile1,
+    backgroundColor: COLORS.greySubtile1,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->